components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/internal/OAuthComponentServiceHolder.java

```diff
@@ -243,4 +245,31 @@ public void setOrganizationUserResidentResolverService(
 
         this.organizationUserResidentResolverService = organizationUserResidentResolverService;
     }
+
+    /**
+     * Register access token response handlers for modifying token response attributes.
+     *
+     * @param accessTokenResponseHandler {@link AccessTokenResponseHandler} instance
+     */
+    public void addAccessTokenResponseHandler(AccessTokenResponseHandler accessTokenResponseHandler) {
```

Comment:
add a new line after method signature. Fix in all places

Reply:
fixed in f2bda44b173e8e1643ee4cc0e2b72b2dd02bead3